### PR TITLE
Options Menu for lua Cvars

### PR DESCRIPTION
**Pull request by alufolie91**
https://git.do.srb2.org/STJr/SRB2/-/merge_requests/2380/diffs?commit_id=75d0d324852b3a2d78278c0471d0164a5ea8605b

Modified a little to mark netvars red

### CODE REVIEW

File: src/lua_consolelib.c [hunk 1]
```diff
@@ -385,13 +396,17 @@ static int lib_cvRegisterVar(lua_State *L)
...
+					if (fasticmp(strvalue, "MIN"))
+						hasminmax = true;
+					cvpv[i].strvalue = strvalue;
 					cvpv[i].value = (INT32)lua_tonumber(L, 6);
 					i++;
 					lua_pop(L, 1);
 				}
 				cvpv[i].value = 0;
 				cvpv[i].strvalue = NULL;
+				hasminmax = false;
```
Comment (Indev450) [inline src/lua_consolelib.c]: Is this line intended? It will cause `hasminmax` always be false

Comment (alufolie91) [reply src/lua_consolelib.c]: uhhhhhhhhhhhhhh somehow it works but ye shouldnt be there lul

Comment (alufolie91) [reply src/lua_consolelib.c]: oh wait it didnt work lmfao oops

Comment (alufolie91) [reply src/lua_consolelib.c]: hm tbh with it fixed, i dont really like the sliders

Comment (alufolie91) [reply src/lua_consolelib.c]: so uhhhhhh, personally id just let it not be a slider

Comment (Indev450) [reply src/lua_consolelib.c]: blobcatpat

**Pull request merged**